shim.go

```diff
@@ -20,59 +20,43 @@ import (
 	"os/exec"
 
 	vc "github.com/containers/virtcontainers"
+	"github.com/golang/glog"
 )
 
-// ShimInfo gathers information needed by the shim.
-type ShimInfo struct {
-	Path  string
-	Token string
-	IP    string
-	Port  string
-}
-
-func startShim(pod *vc.Pod) (int, error) {
-	containers := pod.GetContainers()
-
-	if len(containers) != 1 {
-		return -1, fmt.Errorf("Container list from pod is wrong, expecting only one container")
+func startShim(process *vc.Process, url string) (int, error) {
+	if process.Token == "" {
+		return -1, fmt.Errorf("Token cannot be empty")
 	}
-	container := containers[0]
 
-	shimInfo, err := getShimInfo(container)
-	if err != nil {
-		return -1, err
+	if url == "" {
+		url = defaultProxyURL
 	}
+	glog.Infof("Proxy URL: %s\n", url)
 
 	cmd := exec.Cmd{
-		Path: shimInfo.Path,
-		Args: []string{"-t", shimInfo.Token, "-s", shimInfo.IP, "-p", shimInfo.Port},
+		Path: defaultShimPath,
```

Comment:
This should be a function argument as well. If our config files specified a shim path that's different than `defaultShimPath` then this is not going to work.
And yes, that means we need to load the config file for `exec` as well.